src/signals/incident-management/containers/IncidentDetail/components/Detail/components/ExtraProperties/index.test.tsx

```diff
@@ -235,4 +235,53 @@ describe('<ExtraProperties />', () => {
       /^In huis$/
     )
   })
+
+  it('should handle null and undefined values in the answer property', () => {
+    const items = [
+      {
+        id: 'extra_bedrijven_horeca_terrassen',
+        label: 'Oorzaak overlast',
+        answer: null,
+        category_url:
+          '/signals/v1/public/terms/categories/overlast-bedrijven-en-horeca/sub_categories/overlast-terrassen',
+      },
+      {
+        id: 'extra_bedrijven_gezien',
+        label: 'Heeft u het gezien?',
+        category_url:
+          '/signals/v1/public/terms/categories/wegen-verkeer-straatmeubilair/sub_categories/straatverlichting-openbare-klok',
+        answer: undefined,
+      },
+    ]
+
+    // this test is about handling unexpected values, therefore typing cannot be applied
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Shouldn't the type be changed then?

Reply:
Then it should be changed to any, or not? What's the use of typing if it does not indicate the type a const should be?

Reply:
Not to `any`, but to a type that allows the `answer` prop to be either `null`, `undefined` or whatever it is now.

Reply:
sure